### PR TITLE
Rehack tweaks

### PR DESCRIPTION
**Pull request by lachlanjc**
Before:
screen shot 2018-12-05 at 11 09 37 pm

After:
screen shot 2018-12-05 at 11 10 03 pm

But it's fancier than that, try the demo :)

### CODE REVIEW

**Comment by zanedb** [comment]
be9936a3-2145-41bd-8e33-f9eeab067b7d
f8844076-0a53-4069-859b-4931932c19c2

File: src/components/workshops/Carousel.js [hunk 1]
```diff
@@ -96,7 +104,9 @@ const EmptyProject = emptyProject => (
 
 const ShortList = projects => (
   <StaticWrapper>
-    <CarouselProject project={emptyProject} />
+    {projects.map(project => (
+      <CarouselProject project={project} key={project.screenshot.id} />
+    ))}
```
Comment (polytroper) [inline src/components/workshops/Carousel.js:117]: Lol this was a major oopsie-daisy. Thanks for catching that.

File: src/components/workshops/CarouselProject.js [hunk 2]
```diff
@@ -42,21 +45,26 @@ const LinkBar = styled(Flex).attrs({
...
+  color: 'muted'
+})`
+  text-decoration: line-through;
+`
 
-const AuthorLabel = styled(Label).attrs({
+const AuthorLabel = styled(Text).attrs({
   mx: 1,
-  fontSize: 3
+  fontSize: 3,
```
Comment (polytroper) [inline src/components/workshops/CarouselProject.js:54]: `fontSize={[2, null, 3]}` maybe? Given the crowded/clipped text in Zane's screenshots

**Review by polytroper — APPROVED** [review]
👍

**Pull request merged**